### ISSUE TITLE
如何切换协议？

### ISSUE THREAD

**Issue by wayne1157** [ofzorzo/tcp-fairness-analysis#1]
请问如何切换不同协议，是在自身的linux系统自行设定吗？如何在不同终端设定不同协议？

**Comment by ofzorzo** [comment]
Google tells me you're asking if it is possible to use other protocols. Is that so? If yes, it is not possible. I was interested in analyzing TCP only, so that's what my code does.

If you want to analyze UDP's fairness, a good start would be to mess with the sockets a little bit ([here](https://github.com/ofzorzo/tcp-fairness-analysis/blob/065c9fc484c17645c528698e18b6e696b310fa77/server.py#L48) and [here](https://github.com/ofzorzo/tcp-fairness-analysis/blob/065c9fc484c17645c528698e18b6e696b310fa77/client.py#L11)). I guess you only need to change a few lines; I can try to help you when I wake up.

**Comment by wayne1157** [comment]
Yeah,I want to know if your code support that we compare different TCP protocols,just like cubic and bbr. I have tried to open different terminals to use different protocols.But it seemed that I failed.

**Comment by wayne1157** [comment]
Because TCP protocol has different versions just like TCP tahoe,TCP reno,TCP vegas,bic,cubic,bbr and so on,using your code, I can change the protocol in my ubuntu system.But it is a trouble that I use it to compare different protocols.For example, I set cubic in one terminal and start the client in the terminal;then open another terminal and change the protocol to bbr and start the client.The flow in the first terminal will use bbr,but not cubic.It troubles me a lot.
